packages/web-components/fast-foundation/src/calendar/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./calendar";
+export * from "./calendar.template";
+export * from "./date-formatter";
```

Comment:
I don't think we need to export `date-formatter` as it looks to be a utility specifically intended for the calendar base class and not a public api?

Reply:
I will need it for the date-picker as well. But I can pull that in through a file path as it will only be needed in fast-foundation.

Reply:
When I remove this, there are lots of warnings about missing exports. Part of it is that all of the types that Calendar relies on.